src/routes/engagement_list/engagement_list.tsx

```diff
@@ -1,19 +1,38 @@
 import React from 'react';
-import { Engagement } from "../../schemas/engagement_schema";
+import { Engagement } from '../../schemas/engagement_schema';
 import { EngagementListItem } from './engagement_list_item';
+import {
+  EmptyState,
+  EmptyStateIcon,
+  Title,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { CubesIcon } from '@patternfly/react-icons';
 
 export interface EngagementListProps {
   filter?: (engagement: Engagement) => boolean;
   title?: string;
 }
 
 export function EngagementList({ engagements }: { engagements: Engagement[] }) {
-
+  if (!engagements || !engagements.length) {
+    return (
+      <EmptyState>
+        <EmptyStateIcon icon={CubesIcon} />
```

Comment:
This is an empty state for the search results, right? What do you think if we use a "search icon" instead of cube one? And maybe a primary button for calling to action "Create New"?
I'm getting inspired by [EmptyState Patterfly](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/empty-state#main-content) - Section Filtering or search situation